index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+
+
+  <header id="header ">
+
+    <div id="header-img">
+      <img src="microsoft.png" alt="">
+    </div>
```

Comment:
It is good to give the 'alt' a related value so that if due to any reason the image is not loaded the user can know what kind of image it was.
-instead of 'div' use 'Header' Tag.

Reply:
nice comments @caasperr, you just need to explain further why the header tag is preferable in this line of code, the will help the developer understand why you are requesting for changes and learn from you.

you can say it best practice to use html5 semantics tag such as a header tag instead of a div tag, this will improve SEO and help screen readers to navigate your webpage.

Reply:
Thankyou @xtrahuman i'll take care next time !